### ISSUE TITLE
Fix inspections around tags closing with /}

### ISSUE THREAD

**Issue by mesour** [nette-intellij/intellij-latte#133]
For example, if an unknown tag is closed by `/}`, it says `Malformed tag, missing closing }`, but it has to say `Unknown tag {tagName}`

**Comment by jaroslavlibal** [comment]
I am sorry if my report is not related to this issue, but I hope it is.

Some time ago we investigated (in the first version of your plugin updates) why my IDE freezes on a larger files.

We have come to a conclusion that it freezes due to the unclosed unpaired variant of the label macro. I have added the closing slashes to my code and everything worked fine.

However, the current 1.1.1-RC2 version probably behaves differently.

The closed variant is incorrectly formatted, for example, inside the condition. Plugin says "Malformed tag. Missing closing }":
```
{if $foo}
<div>
     {label bar /}
</div>
{/if}
```

Unclosed variant is well formated:
```
{if $foo}
     <div>
         {label bar}
     </div>
{/if}
```
